projects/Mallard/src/theme/typography.ts

```diff
@@ -1,18 +1,44 @@
-import { Platform } from 'react-native'
+const families = {
+    sans: 'GuardianTextSans-Regular',
+    text: 'GuardianTextEgyptian-Reg',
+}
+
+type FontFamily = keyof typeof families
 
-/**
- * Just the font names.
- *
- * The various styles of fonts are defined in the <Text /> component.
- */
-export const typography = {
-    /**
-     * The primary font.  Used in most places.
-     */
-    primary: Platform.select({ ios: 'Montserrat', android: 'Montserrat' }),
+/*
+Think of these as ems
+*/
+
+const scale = {
+    sans: {
+        [0.5]: {
```

Comment:
I left this side of the scale a bit more loose as the goal is to kinda approximate the intent of the design rather than using actual ems (if we were to use exact units then we wouldn't need a scale!). So .5 is the small text, 1 is body copy, 1-2 is for standfirsts, large body copy and such and 2+ is headings.

Good to check with design on whether it's a good shout to use more exact keys

Reply:
Would it be worth using words here? Small, Regular, Large? That way its a bit more readable and we can be a bit more loose with it?